packages/client/hmi-client/src/utils/github-import.ts

```diff
@@ -0,0 +1,44 @@
+const header = {
+	Authorization: `Bearer ${import.meta.env.VITE_CR_PAT}`
```

Comment:
I don't see where you set this up

Reply:
I just have my own `.env` file in my `/hmi-client`. I don't think we will need one unless there are over 100 requests within an hour or something. I set it up for myself in case

Reply:
Okay, in that case we should remove it, or comment it for now.